easybuild/easyconfigs/h/hwloc/hwloc-1.10.0-GCC-4.9.2.eb

```diff
@@ -0,0 +1,21 @@
+name = 'hwloc'
+version = '1.10.0'
```

Comment:
Sorry guys, v1.10.1 was ready but we got a late AIX bug that I couldn't fix before the hollidays. If I get the required AIX access today, I can do v1.10.1 by the end of the week. Hopefully the admins of these AIX platform will process my account request soon.